x-pack/plugins/monitoring/server/alerts/alerts_factory.test.ts

```diff
@@ -16,31 +24,19 @@ describe('AlertsFactory', () => {
   });
 
   it('should get by type', async () => {
-    const id = '1abc';
     alertsClient.find = jest.fn().mockImplementation(() => {
       return {
         total: 1,
         data: [
           {
-            id,
+            id: ALERT_CPU_USAGE,
           },
         ],
       };
     });
     const alert = await AlertsFactory.getByType(ALERT_CPU_USAGE, alertsClient as any);
     expect(alert).not.toBeNull();
-    expect(alert?.type).toBe(ALERT_CPU_USAGE);
-  });
-
-  it('should handle no alert found', async () => {
```

Comment:
Hm, not sure actually, I think it was giving me problems, and I wanted to see if the suite passes without it. I'll try adding it back

Reply:
@chrisronline I remembered why I removed it now. It's because we don't need to return an empty alert status, since the UI does not care anymore and handles it gracefully:
https://github.com/elastic/kibana/blob/9c39cfb7f96a55aaafdc0e620b8cc7a3ae1ff85a/x-pack/plugins/monitoring/server/lib/alerts/fetch_status.ts#L30-L32

I'll mark this as resolved, but let me know if you think otherwise (and we can add it in a separate issue)

Reply:
Sounds good!